source/plugin/offline-userplayer-data.rst

```diff
@@ -0,0 +1,58 @@
+===================
+Offline Player Data
+===================
+
+It may be necessary for plugins to access player data even when the player is offline.
+One could guess that ``Sponge.getServer().getPlayer()`` returning a ``Player`` can be used for this.
+But since ``Player`` objects only exist for online players, another solution has to be used.
+
+Some plugins store the relevant data themselves and associate the user by using the ``GameProfileManager``.
```

Comment:
I would not say that "some plugins store data themselves" as that is a bad practice that shouldn't be even accidentally enforced. Rather, it should read something like "Rather than using custom code to store player data, plugins can access the _____ manager"